internal/oci/container_freebsd_nocgo.go

```diff
@@ -18,3 +18,15 @@ func getPidStartTime(pid int) (string, error) {
 	fields := strings.Fields(string(data))
 	return fields[7], nil
 }
+
+// getPidData reads the kernel's /proc entry for various data.
+func getPidData(pid int) (*StatData, error) {
+	startTime, err := getPidStartTime(pid)
+	if err != nil {
+		return nil, err
+	}
+	return &StatData{
+		StartTime: startTime,
+		State:     "not implemented",
```

Comment:
could be a follow up, but it seems freebsd uses status as well: https://man.freebsd.org/cgi/man.cgi?query=proc&apropos=0&sektion=5&manpath=CentOS+7.1&arch=default&format=html the location is just slightly different. Could maybe just have platform set `statStateLocation`

Reply:
This man page is for centos... The freebsd [man page](https://man.freebsd.org/cgi/man.cgi?query=procfs&apropos=0&sektion=0&manpath=FreeBSD+13.2-RELEASE&arch=default&format=html) is lacking a pid state.

Reply:
lmao so silly it would even list that as an option